casper/src/main/resources/PoS.rhox

```diff
@@ -344,17 +405,29 @@ new  PoS,
           }
         } |
 
-        contract deposit (@fromUserPk, @amount, returnCh) = {
+        contract deposit (@fromUserPk, @amount, @activeValidators, returnCh) = {
           new vaultCh,
               revAddressCh,
+              indexSender,
+              getBlockData(`rho:block:data`), blockDataCh,
               authKeyCh in {
             @RevVault!("deployerAuthKey", *authKeyCh) |
             for (@authKey <- authKeyCh) {
               revAddressOps!("fromPublicKey", fromUserPk, *revAddressCh) |
+              getBlockData!(*blockDataCh) |
```

Comment:
Again, this would be more readable if the produce on `getBlockData` was in the same scope as the consume.